portal/src/texts/profile/picture.md

```diff
@@ -1,5 +1,286 @@
-## Bilder
+### Hovedprinsipper
+### Persongalleri
+### Omgivelser
+### Kategoribilder
+### Lys og farger
+### Komposisjon
+### Bevegelse
+### Implementering
+### Bildearkiv
+
+## Hovedprinsipper
 
 Fremtind har en todelt fotostil. Vi står enten på toppen av tinden, med full oversikt, eller på bakkeplan sammen med menneskene.
 
-Inntrykket skal være ærlig, ekte og upolert. Vi legger vekt på det hverdagslige og jordnære, for å gi kundene noe de kan relatere seg til. Fordi fargene vi bruker er nøkterne og stilrene, vil fotografier utdype identiteten til Fremtind med varme og følelser.
+**bilde: rettovenfra + bilde: rett forfra**
+
+Fotostilen er ærlig, ekte og upolert. Den er hverdagslig og jordnær, nettopp for å gi kundene våre noe de kan relatere seg til.
+
+Fordi fargene vi bruker er nøkterne og stilrene, vil fotografier utdype identiteten til Fremtind med varme og følelser. Bruk gjerne bilder som henter opp farger fra fremtinds støttefargepallett.
```

Comment:
Forslag: "bruker vi fotografier for å tilføre varme og følelser"

Sløyfe ref til Fremtind siden det er etablert i konteksten

Reply:
Takk, endret det